libs/payments/stripe/src/lib/stripe.utils.ts

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import {
+  StripeApiList,
+  StripePlan,
+  StripeSubscription,
+} from './stripe.client.types';
+
+/**
+ * Returns array of customer subscription plans
+ */
+export async function getSubscribedPlans(
+  subscriptionList: StripeApiList<StripeSubscription>
+) {
+  const subscriptions = subscriptionList.data;
+  if (!subscriptions) return [];
+  return subscriptions
+    .flatMap((sub) => sub.items.data)
+    .map((item) => item.plan as StripePlan);
```

Comment:
This cast should be fixed within the Stripe client typings library. Line 200 of stripe.client.types.ts needs to be changed to NegotiateExpanded with the associated args.